ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -260,8 +280,18 @@ def configure_encryption(self):
             )
         self.do_click(self.dep_loc["next"], enable_screenshot=True)
 
+    def configure_osd_size(self):
+        """
+        Configure OSD Size
+        """
+        device_size = str(config.ENV_DATA.get("device_size"))
+        osd_size = device_size if device_size in osd_sizes else "512"
+        logger.info(f"Configure OSD Capacity {osd_size}")
+        self.choose_expanded_mode(mode=True, locator=self.dep_loc["osd_size_dropdown"])
+        self.do_click(locator=self.dep_loc[osd_size], enable_screenshot=True)
+
     def verify_operator_succeeded(
-        self, operator="OpenShift Container Storage", timeout_install=300, sleep=20
+        self, operator=OCS_OPERATOR, timeout_install=300, sleep=20
```

Comment:
For 4.9, operator=ODF_OPERATOR should be used, isn't it?

Reply:
this is the default value of method